### ISSUE TITLE
grad_norm过大

### ISSUE THREAD

**Issue by sysu18364010** [LDLINGLINGLING/ParetoQ_for_MiniCPM4#1]
我在使用你的sample data来微调qwen3-1.7B，结果发现了过大的grad_norm，但loss是2.0左右，请问你有遇到类似的情况吗

{'loss': 2.1851, 'grad_norm': 292057776128.0, 'learning_rate': 2e-05, 'epoch': 1.0}
